app/src/main/java/com/github/swent/echo/viewmodels/HomeScreenViewModel.kt

```diff
@@ -192,9 +193,10 @@ constructor(
                         !_filtersContainer.value.pendingChecked &&
```

Comment:
knitpick: there are too many conditions here, personally I would do something like: 
val confirm_full_pending = !_filtersContainer.value.confirmedChecked && !_filtersContainer.value.pendingChecked &&  !_filtersContainer.value.fullChecked
wrapping the conditions in variables and then using them instead in the scope makes it easier to read.